src/plugins/telemetryTable/TelemetryTableConfiguration.js

```diff
@@ -99,16 +99,18 @@ define([
         }
 
         hasColumnWithKey(columnKey) {
-            return _.flatten(Object.values(this.columns))
-                .findIndex(column => column.getKey() === columnKey) !== -1;
+            const flatten = arr => arr.reduce((a, b) => a.concat(b), []);
```

Comment:
If we are using this function more than once in the application, can we make it a utility function and import where needed?

Reply:
done